libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/auth/is-available_email_available.json

```
@@ -0,0 +1,19 @@
+{
+    "request": {
+        "method": "GET",
+        "urlPath": "/is-available/email/",
+        "queryParameters": {
+            "q": {
+                "matches": "^e2eflowtestingmobile@example.com$"
```

Using the same email as we do for the login tests is making those tests fail. Can we use something like this here (and the corresponding code updates)?

```suggestion
                "matches": "^newuser@example.com$"
```

---

Oh, I overlooked this route being needed for the Login Tests. Thanks for catching this!